### ISSUE TITLE
`plot_confusion_matrix` has weird white grid lines with Seaborn

### ISSUE THREAD

**Issue by reiinakano** [reiinakano/scikit-plot#13]
As pointed out by @frankherfert , when Seaborn is used (`import seaborn`) with scikit-plot, the confusion matrix tends to have weird white grid lines. Suggestions on how to get rid of this (especially from experienced Seaborn users) without adding a Seaborn dependency would be much appreciated.

![confmatrixseaborn](https://cloud.githubusercontent.com/assets/18363734/23304239/a9da5484-fad3-11e6-88f9-a1a36aa3113d.png)

**Comment by david1309** [comment]
Hello @reiinakano ,

you might try turning 'off' the grid in the plot.

```python
clf.plot_confusion_matrix(x_train, y_train, normalize=True)
plt.grid('off')
```

**Comment by reiinakano** [comment]
Hi @david1309 , I just tried this out, and you're right, it does solve the problem!

Finally this issue can be put to rest. Thanks a lot!

**Comment by reiinakano** [comment]
fixed in #27

**Comment by flicht** [comment]
`plt.grid(False)` worked for me
